CHANGELOG.md

```diff
@@ -1,7 +1,39 @@
-### main
+### 2.1.0
 
-* Bump Maps SDK to 11.5.0.
+* Add ModelLayer API.
+* Support for offline map, allowing users to download and store map data on their devices for use in environments with limited or no internet connectivity.
+* Layer expressions support. Specify expressions when constructing a layer with all new expression support for layers.
+*Before:*
+```dart
+mapboxMap.style.setStyleLayerProperty("layer", "line-gradient",
+    '["interpolate",["linear"],["line-progress"],0.0,["rgb",255,0,0],0.4,["rgb",0,255,0],1.0,["rgb",0,0,255]]');
+
+```
+*After:*
+```dart
+LineLayer(
+  ...
+  lineGradientExpression: [
+    "interpolate",
+    ["linear"],
+    ["line-progress"],
+    0.0,
+    ["rgb", 255, 0, 0],
+    0.4,
+    ["rgb", 0, 255, 0],
+    1.0,
+    ["rgb", 0, 0, 255]
+  ],
+);
+```
+* Expose `text-occlusion-opacity`, `icon-occlusion-opacity`, `line-occlusion-opacity`, `model-front-cutoff`, `lineZOffset` as experimental.
+* Add min/max/default values for most of the style properties.
+* Expose `clusterMinPoints` property for `GeoJSONSource`.
+* Expose `SlotLayer` and `RasterParticleLayer`.
+* Expose `LocationComponentSettings.slot`.
+* Add `@experimental` annotation to relevant APIs.
 * Expose `LineJoin.NONE`.
```

Comment:
JIRA said this should be in beta, not this stable release. Should we add it here?

Reply:
This change is scheduled for 2.1.0 stable.